src/algebra/direct_limit.lean

```diff
@@ -29,24 +36,30 @@ variables {ι : Type v}
 variables [dec_ι : decidable_eq ι] [directed_order ι]
 variables (G : ι → Type w)
 
-/-- A directed system is a functor from the category (directed poset) to another category.
-This is used for abelian groups and rings and fields because their maps are not bundled.
-See module.directed_system -/
+/-- A directed system is a functor from the category (directed poset) to another category. -/
```

Comment:
```suggestion
/-- A directed system is a functor from a category (directed poset) to another category. -/
```

Reply:
I have no idea what this comment is trying to say, but will assume that you do!